projects/ingress/pkg/setup/setup_syncer.go

```diff
@@ -31,6 +33,8 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const defaultClusteringressProxyAddress = "clusteringress." + gloodefaults.GlooSystem + ".svc.cluster.local"
```

Comment:
Shouldn't this be
```suggestion
const defaultClusteringressProxyAddress = "clusteringress-proxy." + gloodefaults.GlooSystem + ".svc.cluster.local"
```
?